src/main/features/dashboard/views/status/claimant/init.njk

```diff
@@ -0,0 +1 @@
+{{ t('Awaiting Response..') }}
```

Comment:
do we need two periods here?

Reply:
This is never going to display to the customer as we and when we finish all the transitions for part admit, full admit and full defence the state will not going to be init for any claim .... this is just for testers who can see this as and when we finish the each of part admit, full admit and full defence. even though I will remove those dot's

Reply:
its done, committed the latest changes